src/lib/browse/schemes/SchemesLayer.svelte

```diff
@@ -30,29 +35,47 @@
 
   let errorMessage = "";
 
-  let atfName = "atf_schemes";
-  let atfTitle = "ATF schemes";
-  let atfShow = showHideLayer(atfName);
-  let atfStyle = "fundingProgramme";
-  $: [atfColor, atfLegend] = pickStyle(atfStyle);
+  let mainAtfName = "main_atf_schemes";
```

Comment:
There's some boilerplate copying here. We could consider abstracting it a bit, but it has to be done carefully to be useful (like bundling all these variables together, leaving room to customize names/labels, etc). I'd like to decide about having one local layer vs local ATF + local LCWIP before doing anything though.